sheetstorm/tests/test_unit_generate_test_data.py

```diff
@@ -0,0 +1,207 @@
+from dateutil.relativedelta import relativedelta
+from django.test import TestCase
+from django.utils import timezone
+from parameterized import parameterized
+
+from managers.factories import ProjectFactory
+from managers.models import Project
+from sheetstorm.management.commands.generate_test_data import Command
```

Comment:
Mostly it is wrong behavior to change name of imported package/class/function but in this case I think it will be more readable and meaningful to use different name than just `Command`. Please change it to use different name like "GenerateTestDataCommand". I think it will be more friendly name :)

Reply:
Done